patterns/direct_requestor_blocking.py

```diff
@@ -0,0 +1,121 @@
+## Goal: A simpe request pulblisher to publish a request with reply_to set to any topic.
+# Unlike regular publisher, this will be built on RequestReplyMessagePublisher which exposes the ability to set 'Reply To' field
+import os
+import platform
+import time
+import calendar;
+
+# Import Solace Python  API modules from the solace package
+from solace.messaging.messaging_service import MessagingService, ReconnectionListener, ReconnectionAttemptListener, \
+            ServiceInterruptionListener, RetryStrategy, ServiceEvent
+from solace.messaging.resources.topic import Topic
+from solace.messaging.errors.pubsubplus_client_error import PubSubPlusClientError
+
```

Comment:
I would recommend importing the RequestReplyMessagePublisher
`from solace.messaging.publisher.request_reply_message_publisher import RequestReplyMessagePublisher`

Reply:
Done